app/overnight/auto-update-statuses.js

```diff
@@ -10,8 +12,13 @@ const autoUpdateStatuses = async () => {
   try {
     const today = new Date()
 
+    const insuranceRevertAlreadyRun = await hasJobRunBefore('Success Revert In-breach Insurance to Exempt')
+
     await sequelize.transaction(async (t) => {
       result = result + await setExpiredCdosToFailed(today, user, t) + ' | '
+      if (!insuranceRevertAlreadyRun) {
+        result = result + await revertExpiredInsurance(today, user, t) + ' | '
+      }
       result = result + await addBreachReasonToExpiredInsurance(today, user, t) + ' | '
```

Comment:
as you're going to need to redo this branch, any chance in putting a line break below and above this?